runtime/themes/ayu_dark.toml

```diff
@@ -33,11 +33,15 @@
 
 # Interface
 "ui.background"= { bg = "background" }
-"ui.cursor" = { fg = "dark_gray", bg = "orange" }
+"ui.cursor" = { fg = "dark_gray", bg = "yellow" }
+"ui.cursor.primary" = { fg = "dark_gray", bg = "orange" }
```

Comment:
Maybe do the same for ayu light and ayu mirage? I don't think this is done in other editors but maybe it is useful since we need to differentiate it.

Reply:
Done